contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,155 @@
+    2. Checkout the current repo (using `actions/checkout`)
+    3. Install dependencies `npm ci`
+    4. Build `npm run build --if-present`
+    5. Run tests `npm run test`
+    6. Report the status of the workflow via a status badge
+    ![](images/status.png)
+5. Since we set the workflow up on the `master` branch, when we commit the newly-created workflow file, we can see GitHub Actions in action by navigating to the Actions tab
+    ![](images/progress.png)
+
+---
+
+## Where to Go From Here
+<!--
+Give more resources. Instead of listing a lot of links, provide a brief summary of what value each resource can provide the reader.
+-->
+The official [documentation](https://help.github.com/en/actions) is a good resource to learn more about GitHub Actions.
+The [starter workflows](https://github.com/actions/starter-workflows) repository contains good examples of 
+CI workflows for popular languages as well as automation workflows for GitHub.
+Here are a list of [awesome actions](https://github.com/sdras/awesome-actions) that you can use when writing your workflow.
+ 
+
+A powerful extension to GitHub Actions is to use the [github-script](https://github.com/actions/github-script) action 
+to run JavaScript within the workflow using GitHub's [Octokit](https://octokit.github.io/rest.js/v17/) API. 
+The Octokit API allows you to create workflows that span across GitHub's ecosystem of services.
+
+I have also compiled a list of popular repositories that use GitHub Actions that you can use as a reference. Do note that this is far from a comprehensive list and is more of a sample of how organizations are leveraging GitHub Actions in their development and production workflow.
+
+### Repositories That Use GitHub Actions
+
+Org/Project | Workflows | Notes
+--- | --- | ---
+Bootstrap | [test](https://github.com/twbs/bootstrap/blob/master/.github/workflows/test.yml) | Comprehensive set of tests including custom tests and coveralls
+Oh My Zsh | [test](https://github.com/ohmyzsh/ohmyzsh/blob/master/.github/workflows/main.yml) | Workflow uses mainly shell commands
+React Native | [needs-attention](https://github.com/facebook/react-native/blob/master/.github/workflows/needs-attention.yml), [process-label-actions](https://github.com/facebook/react-native/blob/master/.github/workflows/process-label-actions.yml) | Automatically apply labels to newly created issues and adds comments to issues based on labels
+Node.js | [workflows](https://github.com/nodejs/node/tree/master/.github/workflows) | Linters, tests and build jobs
+Chart.js | [CI](https://github.com/chartjs/Chart.js/blob/master/.github/workflows/ci.yml) | Conditional workflow for windows and ubuntu
+Next.js | [workflows](https://github.com/zeit/next.js/tree/canary/.github/workflows) | Complex yet well structured build/test/deploy workflow. Automatically reports bundle size statistics in pull requests and on releases. Scheduled tests that run every 12 hours.
```

Comment:
I don't think this section adds much value, or can be replaced with just one sentence.

Reply:
This entire section? I think that readers may learn more from examples of GitHub Actions used in production compared to sandboxed examples such as the one above.

Reply:
It's fine if they are supposed to serve as educational examples. Can keep.